test/pytest/test_hwo_tango_shutter.py

```diff
@@ -0,0 +1,123 @@
+"""Test the HardwareObjects.TangoShutter shutter hardware object."""
+
+from typing import Callable
+
+import gevent
+import pytest
+from gevent import Timeout
+from tango import (
+    DeviceProxy,
+    DevState,
+)
+from tango.server import (
+    Device,
+    command,
+)
+from tango.test_context import DeviceTestContext
+
+from mxcubecore.HardwareObjects import TangoShutter
+
+TANGO_SHUTTER_STATES_MAPPING = """
+{"OPEN": "OPEN", "CLOSED": "CLOSE", "MOVING" : "MOVING"}
+"""
+
+
+class Shutter(Device):
+    """Very simple Tango shutter device, that only goes
+    between 'open' and 'close' states."""
+
+    def __init__(self, *args, **kwargs):
+        self._is_open = False
+        super().__init__(*args, **kwargs)
+
+    @command()
+    def Open(self):
+        self._is_open = True
+
+    @command()
+    def Close(self):
+        self._is_open = False
```

Comment:
Instead of disabling N802 check for this whole file in [ruff.toml](https://github.com/mxcube/mxcubecore/pull/1158/files#diff-26f436cb29eecc7bb4f9066747b43df8da81ac808955ce95c109f5d7aa778989),
disable it only for these two special case methods, like this:

```suggestion
    @command()
    def Open(self):  # noqa: N802
        self._is_open = True

    @command()
    def Close(self):  # noqa: N802
        self._is_open = False
```

Reply:
Yes, I was about to comment on this as well.

Otherwise:

```suggestion
    @command()
    def open(self):
        self._is_open = True

    @command()
    def close(self):
        self._is_open = False
```

The upper case "convention" for Tango commands does not really matter here in a test case, I guess. Then you would need to change in the other places in the test case of course.

Reply:
Done